devdox/app/exceptions/handlers.py

```
@@ -0,0 +1,121 @@
+        exc_type,
+        path,
+        method,
+        status_code,
+    )
+
+    debug_payload = None
+    if settings.API_ENV in ["development", "test"]:
+        debug_payload = {"exception": type(exc).__name__, "str": str(exc)}
+
+    return APIResponse.error(
+        message=app.exceptions.exception_constants.SERVICE_UNAVAILABLE,
```

Pylint (3.3.7)

[info] Module 'app' has no 'exceptions' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects.

(I1101)